beacon/client/client.go

```diff
@@ -19,8 +19,9 @@ import (
 )
 
 const (
-	BeaconEventTimeout = structs.DurationPerSlot + (structs.DurationPerSlot / 4)
-	BeaconQueryTimeout = 20 * time.Second
+	BeaconEventTimeout       = structs.DurationPerSlot + (structs.DurationPerSlot / 4)
```

Comment:
Shouldn't we make is configurable, not to have another PR tomorrow ? :) The "4" i mean

Reply:
The BeaconEventTimeout or the BeaconConsecutiveTimeout?

Reply:
To be fair both look like good candidates, as we know that's different on each one of 3 chains

Reply:
Nice, I will